src/orchard/namespace.clj

```diff
@@ -135,6 +167,46 @@
        (filter identity)
        (classpath-namespaces)))
 
+(defn project-jvm-ns-forms
```

Comment:
Isn't much of this ns JVM specific anyways? Seems weird to use a different naming for just a single function.

Reply:
IMO `project-namespaces` should have been named `project-jvm-namespaces`. In the end it's just a tweak away from being cljs-compatible.

I'd propose to deprecate `project-namespaces` in favor of the alternative, laying out the path forward.

Reply:
Or it can just a be a `orchard.java.namespace` (similar to some namespaces in Clojure and other projects). I don't like including implementation details in the names of functions and introducing internal naming inconsistencies. I understand your point, but given there's no ongoing push to add ClojureScript support this seems like a case of premature optimization.

Reply:
For now I'll leave things are they were before, even if they weren't ideal.

We can refactor later indeed, driven by a specific need.